### ISSUE TITLE
Setting email / phone label when adding new person doesn't work

### ISSUE THREAD

**Issue by jonaspiela** [pipedrive/client-nodejs#248]
It looks as if the nodejs library doesn't correctly translate javascript objects to json. Check this code:

```
const object = {
    name: `John Doe`,
    email: [{
      value: "some@example.org",
      label: "work",
      primary: true,
    }],
    phone: [{
      value: "+49 123 456789",
      label: "work",
      primary: true,
    }],
  };

const personsApiOpts = pipedrive.NewPerson.constructFromObject(object);
await personsApiInstance.addPerson(personsApiOpts)
```

When adding a new person with the `object` as above, it doesn't seem to recognize the email / phone objects but instead adds `[Object object]`. This is what the API returns as a result that was added. This is also reflected in pipedrive itself:
```
email: [{
  label: ""
  primary: true
  value: "[object Object]"
}]
```

When running this through postman with the following body request, it works:

```
{
    "name": "test #3",
    "owner_id": "<integer>",
    "org_id": "207",
    "email": [{
        "value":" test@example.com",
        "label": "work",
        "primary": true
    }],
    "phone": [
        "<string>",
        "<string>"
    ],
    "visible_to": "<string>",
    "add_time": "<string>"
}
```

It responds with
```
"email": [
            {
                "label": "work",
                "value": "test@example.com",
                "primary": true
            }
        ],
```

**Comment by dmitriyeff** [comment]
@jonaspiela hi! Thank you for sharing this info. Indeed, there is a bug. I will come back once it is fixed.

**Comment by dmitriyeff** [comment]
Issue is fixed #255, you can test it now.

@jonaspiela , thank you one more time for pointing this out.
